eng/docs/api/assets/README.md

```diff
@@ -0,0 +1,255 @@
+# Doxygen Template for C++ API Documentation
+
+When upgrading Doxygen, the following customizations have been made. 
+
+## Export Doxygen Templates
+
+Doxygen templates have several parts:
+
+1. Doxygen Layout file `DoxygenLayout.xml`
+1. Template files:
+    1. `header.html`
+    1. `footer.html`
+    1. `style.css` (renamed from `customdoxygen.css`)
+
+Further documentation on customizing Doxygen can be found here: https://www.doxygen.nl/manual/customize.html
+
+To export the layout file:
+
+```powershell
+cd eng\docs\api\assets
+doxygen -l
+```
+
+To export the HTML/CSS files:
+
+```powershell
+cd eng\docs\api\assets
+doxygen -w html header.html footer.html style.css
+```
+
+## Configure templates
+
+### DoxygenLayout.xml
+
+Around line 6, change:
+
+```xml
+<tab type="pages" visible="yes" title="" intro=""/>
+```
+
+To read:
+
+```xml
+<tab type="pages" visible="yes" title="Concepts and Related Pages" intro=""/>
+```
+
+### header.html
+
+In the `<head>` of the page add the following snippets:
```

Comment:
Can you explain the "why" for some of these required changes, to help readers like me understand the motivation behind them (especially in the future).